src/main/java/seedu/us/among/model/endpoint/UniqueEndpointList.java

```diff
@@ -42,9 +42,9 @@ public boolean contains(Endpoint toCheck) {
      */
     public void add(Endpoint toAdd) {
         requireNonNull(toAdd);
-        if (contains(toAdd)) {
-            throw new DuplicateApiEndpointException();
-        }
+        // if (contains(toAdd)) {
```

Comment:
I think we should discuss more if we should allow duplicate endpoints because other TA/tester may view it as a bug.

Reply:
To add on, I think it's possible to check for duplicates but in this case every single field (including tags) would need to match (unlike just name and address in the original AB3).

Reply:
Once again, duplicate endpoints have been considered to issues solved